.github/workflows/test_in_devenv.yml

```diff
@@ -84,6 +89,48 @@ jobs:
               exit 1
             fi
 
+      - name: Test CUDA Quantum MPI4Py Plugin
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd $CUDAQ_REPO_ROOT
+            python3 -m pip install mpi4py~=3.1
+            rm -f build/lib/plugins/libcudaq-comm-plugin.so
+            ctest --test-dir build -R MPIApiTest -V
+            mpi4py_status=$?   
+            if [ ! $mpi4py_status -eq 0 ] ; then
+              echo "::error file=test_in_devenv.yml::Test CUDA Quantum MPI4Py Plugin failed with status $mpi4py_status."
+              exit 1
+            fi
+      
+      - name: Test CUDA Quantum MPI Plugin Activation
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd /usr/local/cudaq/distributed_interfaces/ 
+            # Set MPI_PATH depending on OMPI/MPICH
+            has_ompiinfo=$(which ompi_info || true)
+            if [[ ! -z $has_ompiinfo ]]; then
+              export MPI_PATH="/usr/lib/$(uname -m)-linux-gnu/openmpi/"
+            else
+              export MPI_PATH="/usr/lib/$(uname -m)-linux-gnu/mpich/"
+            fi
+            # Run the activation script
+            source activate_custom_mpi.sh
+            echo $CUDAQ_MPI_COMM_LIB
+            # Rerun the MPI plugin test
+            cd $CUDAQ_REPO_ROOT
+            ctest --test-dir build -R MPIApiTest -V
```

Comment:
Is it intentional that the build directory also contains build/lib/plugins/libcudaq-comm-plugin.so?

Reply:
Yes, it is intentional. This is the built-in plugin, which is built when [MPI is detected at compile time](https://github.com/NVIDIA/cuda-quantum/blob/942267d549dd1fc4b4ae6eea024191d42f8b33cb/runtime/cudaq/distributed/builtin/CMakeLists.txt#L9-L19). This `build/lib/plugins/libcudaq-comm-plugin.so` file will be installed to `<install_dir>/lib/plugins` (along with the mpi4py plugin).